commons-mfa/src/main/java/org/exoplatform/mfa/api/MfaService.java

```diff
@@ -34,4 +36,13 @@ public boolean isProtectedUri(String requestUri) {
                                .count() > 0;
     
   }
+
+  public boolean canAccess() {
+    UserACL userACL = CommonsUtils.getService(UserACL.class);
+    String[] protectedGroup = System.getProperty("exo.mfa.protectedGroupNavigations") != null ? System.getProperty("exo.mfa.protectedGroupNavigations").split(",") : new String[0];
```

Comment:
if (System.getProperty("exo.mfa.protectedGroupNavigations") != null) {
String[] protectedGroup = System.getProperty("exo.mfa.protectedGroupNavigations").split(",");
 for ....
}